### PR TITLE
feat(issue-41): Add Port to knative

### LINKED ISSUE #41: Support defining a port to expose on the knative manifest
We might need to override the default port exposed by knative.

The user should be able to define it as part of the deploy command or in the config file.

### PR DESCRIPTION
**Pull request by amanvr**
Fixed #41

### CODE REVIEW

File: src/cli/flags.go [hunk 1]
```diff
@@ -41,6 +41,13 @@ func Flags(command FlagsType) []cli.Flag {
 				Required: true,
 				Category: "deploy",
 			},
+			&cli.StringFlag{
+				Name:     "app-port",
+				EnvVars:  []string{"KKA_APP_PORT"},
+				Required: false,
+				Category: "deploy",
+				Value:    "80",
```
Comment (LucaLanziani) [inline src/cli/flags.go:49]: I wouldn't define a default value but if the port is not specified we should fallback to knative behaviour, meaning we should not add the "ports:" section to the yaml

File: assets/knative/service.yaml.tmpl [hunk 2]
```diff
@@ -8,3 +8,5 @@ spec:
     spec:
       containers:
         - image: {{ .RemoteTag }}
+          ports:
+          - containerPort: {{ .Port }}
```
Comment (LucaLanziani) [inline assets/knative/service.yaml.tmpl:12]: This should only be added if port is defined

Comment (LucaLanziani) [reply assets/knative/service.yaml.tmpl:12]:
```suggestion
{{ if .Port }}
          ports:
          - containerPort: {{ .Port }}
{{ end }}
```

Comment (amanvr) [reply assets/knative/service.yaml.tmpl:12]: Fixed

**Pull request closed without merging**